{"status":"OK","subtests":[{"name":"ResizeObserver reports the exception from its callback in the callback's global object","status":"FAIL"}]}